### PR TITLE
Initial content

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss</groupId>
+        <artifactId>jboss-parent</artifactId>
+        <version>37</version>
+    </parent>
+    <groupId>org.jboss.eap.cloud</groupId>
```
Comment (jmesnil) [inline pom.xml]: for consistency with new groupID for EAP XP, this should be `org.jboss.eap.xp.cloud`

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss</groupId>
+        <artifactId>jboss-parent</artifactId>
+        <version>37</version>
```
Comment (jmesnil) [inline pom.xml]: can be bumped to `39`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
 # eap-xp-cloud-galleon-pack
-Galleon feature-pack for EAP XP on the Cloud
+Galleon feature-pack for EAP XP on the Cloud.
+
+WARNING: Until we have an XP artifacts, this implementation is based on upstream WildFly, with an eap-cloud-galleon-pack built against wildfly-ee.
+The eap-cloud-fp based on wildfly is: https://github.com/jfdenise/eap-cloud-galleon-pack/pull/new/deps_on_wildfly_for_xp_mock
+ 
+
+
+The ``org.jboss.eap.cloud:eap-xp-cloud-galleon-pack`` Galleon feature-pack provisions a set of additional features allowing you to configure an \
```
Comment (jmesnil) [inline README.md]: org.jboss.eap.xp.cloud:eap-xp-cloud-galleon-pack

File: doc/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+into the pom.xml file of your project.
+
+The "package" goal of the plugin allows you to:
+
+* Provision an EAP XP server tailored to your needs.
+* Incorporate custom content (keystores, properties files, ...).
+* Execute CLI scripts to configure the EAP XP server.
+* Deploy your application in the provisioned server.
+
+The generated server is then ready to be installed inside a Docker Image to be deployed in your cluster.
```
Comment (jmesnil) [inline doc/index.md:13]: s/Docker Image/container image/

File: doc/index.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+For example:
+
+```xml
+<feature-packs>
+  <feature-pack>
+    <location>org.jboss.eap.xp:wildfly-galleon-pack</location>
+  </feature-pack>
+  <feature-pack>
+    <location>org.jboss.eap.cloud:eap-xp-cloud-galleon-pack</location>
```
Comment (jmesnil) [inline doc/index.md]: will become 

```
 <location>org.jboss.eap.cloud.xp:eap-xp-cloud-galleon-pack</location>
```

**Review by jmesnil — CHANGES_REQUESTED** [review]
looks good to me with the caveat that the groupId should be changed everywhere to `org.jboss.eap.xp`

**Comment by jfdenise** [comment]
@jmesnil , thanks fixed!

**Pull request merged**